aws/grafana-dashboards/README.md

```diff
@@ -0,0 +1,41 @@
+# AWS Managed Grafana Dashboards
+
+Adds the option to create some basic dashboards.
+
+Example:
+
+```terraform
+module "grafana_dashboards" {
+  source = "github.com/thoughtbot/flightdeck//aws/grafana-datashboards?ref=VERSION"
+
+  create_istio_dashboard = true
```

Comment:
what would be the use case of this module if this variable was false?

Reply:
That's a good question; it would do nothing. The idea is that this module will create multiple dashboards (not only istio). We could find a way to ensure that at least one of the variables is true. Otherwise, it's pointless to use this module if all of the variables are false